src/functionalTest/resources/scenarios/repository-connector-redirect-passed-to-end-user/tasks.conf

```diff
@@ -0,0 +1,19 @@
+global.handler.fragmentsHandler.config {
+  tasks {
+    sample-task {
+      action = sample-http // _success
+    }
+  }
+
+  actions {
+    sample-http {
+      factory = http
```

Comment:
Can we use an empty task here? Or simply use some `inline-payload`?

Reply:
Removed this config completely.